### PR TITLE
release 1.0.0 for GA

### PR DESCRIPTION
**Pull request by alexander-fenster**
- CI to be run as non-privileged user to save time
- Bumping version to 1.0.0. Ready for ~~takeoff~~ GA!

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,6 +91,7 @@ jobs:
   node4:
     docker:
       - image: node:4
+        user: node
```
Comment (lukesneeringer) [inline .circleci/config.yml]: Why this change? It is unusual to run things in Docker not as root.

Comment (alexander-fenster) [reply .circleci/config.yml]: To save 5 minutes per CI iteration.
